packages/pentaho/src/Card/Card.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  ExtractNames,
+  fixedForwardRef,
+  PolymorphicComponentRef,
+  PolymorphicRef,
+  useDefaultProps,
+} from "@hitachivantara/uikit-react-core";
+
+import { staticClasses, useClasses } from "./Card.styles";
+
+export { staticClasses as cardClasses };
+
+export type HvCardClasses = ExtractNames<typeof useClasses>;
+
+export type HvCardProps<C extends React.ElementType = "div"> =
+  PolymorphicComponentRef<
+    C,
+    {
+      /** A Jss Object used to override or extend the styles applied. */
+      classes?: HvCardClasses;
+    }
+  >;
+
+/**
+ * The Pentaho Card component.
+ */
+export const HvCard = fixedForwardRef(function HvCard<
+  C extends React.ElementType = "div",
+>(props: HvCardProps<C>, ref: PolymorphicRef<C>) {
+  const {
+    classes: classesProp,
+    className,
+    component: Component = "div",
+    disabled = false,
+    onClick: onClickProp,
+    onMouseDown: onMouseDownProp,
+    children,
+    ...others
+  } = useDefaultProps("HvCard", props);
+
+  const { classes, cx } = useClasses(classesProp);
+  const handleClick: HvCardProps["onClick"] = (e) => {
+    if (disabled) return;
+    onClickProp?.(e);
+  };
+
+  const handleMouseDown: HvCardProps["onMouseDown"] = (e) => {
+    if (disabled) return;
+    onMouseDownProp?.(e);
+  };
+
+  return (
+    <Component
+      ref={ref}
+      className={cx(classes.root, className)}
+      onClick={handleClick}
+      onMouseDown={handleMouseDown}
+      {...others}
+    >
```

Comment:
shouldn't we also pass `disabled` to the component?

Reply:
I'm actually removing the disabled functionality as there's no requests for that feature at all and that would just add up to the confusion. If needed we'll add it later.